test/tests/functional/pbs_trillion_jobid.py

```diff
@@ -144,7 +144,7 @@ def stop_and_restart_svr(self, restart_type):
         self.assertTrue(self.server.isUp(), restart_msg)
 
     def submit_job(self, sleep=10, lower=0,
-                   upper=0, job_id=None, job_msg=None):
+                   upper=0, job_id=None, job_msg=None, status=None):
```

Comment:
Since the function only checks for 'R' state, I'd suggest you rename to status to something like 'running' or 'check_running' and make it a boolean with a default value as 'False'.

Reply:
Addressed!